core/src/main/scala/org/apache/spark/rdd/NewHadoopRDD.scala

```diff
@@ -153,34 +157,19 @@ class NewHadoopRDD[K, V](
           throw new java.util.NoSuchElementException("End of stream")
         }
         havePair = false
-
-        // Update bytes read metric every few records
-        if (recordsSinceMetricsUpdate == HadoopRDD.RECORDS_BETWEEN_BYTES_READ_METRIC_UPDATES
```

Comment:
Okay, that's fine then. I looked and it's all private[spark] so actually there is no change to visibility.